indico/modules/events/abstracts/blueprint.py

```diff
@@ -16,14 +16,18 @@
 
 from __future__ import unicode_literals
 
+from indico.modules.events.abstracts.controllers.display import RHDisplayAbstract
 from indico.modules.events.abstracts.controllers.management import (RHAbstracts, RHManageBOA, RHAbstractList,
                                                                     RHAbstractListCustomize, RHAbstractListStaticURL,
-                                                                    RHManageAbstractSubmission)
+                                                                    RHManageAbstractSubmission, RHManageAbstract)
 from indico.web.flask.wrappers import IndicoBlueprint
 
 _bp = IndicoBlueprint('abstracts', __name__, url_prefix='/event/<confId>', template_folder='templates',
                       virtual_template_folder='events/abstracts')
 
+# Display pages
+_bp.add_url_rule('/abstract/<int:abstract_id>', 'display_abstract', RHDisplayAbstract)
```

Comment:
I'd even go for `/abstracts/<int:abstract_id>/` because we will for sure have actions like `/abstracts/<int:abstract_id>/edit` or `/abstracts/<int:abstract_id>/withdraw`.